bottlecap/src/lifecycle/invocation/processor.rs

```diff
@@ -390,7 +408,8 @@ impl Processor {
             _ = offsets.process_chan_tx.send(());
         }
 
-        // todo(duncanista): Add missing metric tags for ASM
+        context.absorb_appsec_tags();
```

Comment:
I think we could, since this is important part, make this method receive a mutable span which it will add the tags to.

Maybe in the processor, or trace agent, whichever seems fit, we detect if a span is the Lambda span and we absorb the tags there. This would make it work for all runtimes, WDYT?

Reply:
Yes I think this is the way forward, really. And this logic can can probably be moved to the `appsec::Processor` as well.